controls/tests.py

```diff
@@ -895,3 +895,107 @@ def create_simple_import_record(self):
+        # Components and their statements?
+        self.assertEqual(Element.objects.all().exclude(element_type='system').count(), 1)
+        self.assertEqual(Statement.objects.all().count(), 3)
+
+    def test_update_project_json_import(self):
+        """
+        Testing the update of a project through project JSON import and ingestion of components and their statements
+        """
+
+        # login as the first user and create a new project
+        self._login()
+        self._new_project()
+
+        # Checks the number of projects and components before the import
+        self.assertEqual(Project.objects.all().count(), 3)
+        self.assertEqual(Element.objects.all().exclude(element_type='system').count(), 0)
+
+        ## Update currentproject
+        # click import project button, opening the modal
+        self.click_element("#action-buttons\ action-row > div.btn-group > button:nth-child(4)")
+        ## select through the modal information needed and browse for the import needed
+        self.select_option_by_visible_text("#id_appsource_compapp", "project")
+        try:
+            self.select_option_by_visible_text("#id_appsource_version_id", "<project/simple_project(3)>")
+        except:
+            self.select_option_by_visible_text("#id_appsource_version_id", "<project/simple_project(9)>")
+
+
+        #self.browser.find_element_by_id("id_file").click()
+        file_input = self.browser.find_element_by_css_selector("#id_file")
+
+        file_path = os.getcwd() + "/fixtures/test_project_import_data.json"
+        # convert filepath if necessary and send keys
+        self.filepath_conversion(file_input, file_path, "sendkeys")
+        self.browser.find_element_by_id("import_component_submit").click()
+
+        # Check the new number of projects, and validate that it's the same
+        self.assertEqual(Project.objects.all().count(), 3)
+        # Has the updated name?
+        self.assertEqual(Project.objects.get(id=3).title, "New Test Project")
+
+        # Components and their statements?
+        self.assertEqual(Element.objects.all().exclude(element_type='system').count(), 1)
+        self.assertEqual(Statement.objects.all().count(), 3)
+
+    def test_project_json_export(self):
+        """
+        Testing the export of a project through JSON and ingestion of components and their statements
+        """
+
+        # login as the first user and create a new project
+        self._login()
+        self._new_project()
+
+        # export the only project we have so far
+        self.navigateToPage('/systems/3/export')
+
+        # Project title to search for in file names
+        project_title = "I_want_to_answer_some_questions_on_Q._3"
+        file_system = os.listdir(os.getcwd())
+        # Assert there is a file
+        for file_name in file_system:
+            if file_name ==project_title:
+                project_title = file_name
+
+        self.assertIn(file_name, file_system)
```

Comment:
You could also take a look at how the `test_component_download_oscal_json` test works (in the same controls/tests.py file) to check whether or not a specific file was downloaded, rather than looping through an entire directory, just in case that directory gets full and that loop takes a while